src/components/QuranReader/ContextMenu.tsx

```diff
@@ -78,13 +79,20 @@ const ContextMenu = () => {
                   `sidebar_navigation_${isSidebarNavigationVisible ? 'close' : 'open'}_trigger`,
                 );
                 e.stopPropagation();
-                dispatch(setIsVisible(!isSidebarNavigationVisible));
+                if (isSidebarNavigationVisible === 'auto') {
+                  // eslint-disable-next-line no-unneeded-ternary
+                  const shouldBeVisible = isMobile() ? true : false;
```

Comment:
nit: `shouldBeVisible = isMobile() || false;` and no need for `// eslint-disable-next-line no-unneeded-ternary`